chart/templates/hpa-external-metric.yaml

```diff
@@ -18,12 +18,15 @@ spec:
 {{- end }}
 
 {{- if .Values.worker.autoscaling.enabled }}
+{{- $sidekiq_busy := "max:sidekiq.busy{kube_cluster_name:dsva-vagov-" }}
+{{- $sidekiq_processes := "max:kubernetes_state.deployment.replicas_available{kube_cluster_name:dsva-vagov-" }}
+{{- $deployment := "-cluster,kube_deployment:" }}
 ---
 apiVersion: datadoghq.com/v1alpha1
 kind: DatadogMetric
 metadata:
-  name: sidekiq-enqueued-jobs
+  name: sidekiq-utilization
   namespace: {{ $root.Release.Namespace }}
 spec:
-  query: avg:sidekiq.enqueued{env:eks-{{ .Values.target_env }}}
+  query: ({{ $sidekiq_busy }}{{ $root.Values.target_env }}{{ $deployment}}vets-api-sidekiq} / ({{ $sidekiq_processes }}{{ $root.Values.target_env }}{{ $deployment}}vets-api-sidekiq}*12)*100)
```

Comment:
Would be nice if this `12` here could pull from an ENV var or something. Which I think could be possible once we remove Sidekiq Alive.

Reply:
You could make the threads a value in values.yaml instead, so when we do remove sidekiq-alive, we don't have to do a helm chart release

Reply:
Added